source/threads.tex

```diff
@@ -12140,9 +12150,16 @@
 \begin{itemdescr}
 \pnum
 \effects
-As if \tcode{*this = packaged_task(std::move(f))}, where
+Equivalent to:
+\begin{codeblock}
+if (!valid())
+  throw future_error(future_errc::no_state);
+*this = packaged_task(allocator_arg, a, std::move(f));
```

Comment:
The paper does not put a space between `a,` and `std::move`, but I assume that such whitespace slip-ups don't require a fixup commit.